go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -464,7 +504,24 @@ func createNodeManagementPorts(name string, nodeAnnotator kube.Annotator, waiter
 		}
 		config.OvnKubeNode.MgmtPortRepresentor = rep
 	}
-	ports := NewManagementPorts(name, subnets)
+
+	if config.OvnKubeNode.Mode == types.NodeModeDPUHost {
+		link, err := getManagmentPortNetDev(config.OvnKubeNode.MgmtPortNetdev)
+		if err != nil {
+			return nil, nil, err
+		}
+		err = exportManagementPortAnnotation(link, nodeAnnotator)
+		if err != nil {
+			return nil, nil, err
+		}
+	} else if config.OvnKubeNode.Mode == types.NodeModeDPU {
+		vfrep, err := importManagementPortAnnotation(node)
```

Comment:
This code only runs once at boot right?
What if the DPU-Host decides to replace `ovnNodeManagementPort` with a different port. Is there any mechanism to monitor for changes to this annotation?

Reply:
This could be true in the case of using device plugin resources (with a pool of >2 VFs) and the ovnkube-node on the DPU-Host is restarted.

Reply:
Is `checkManagementPortHealth()` periodic polling, if so do you think it would also be appropriate to poll for node annotations?

Reply:
It could. But the problem here is that we need to change interface. I do remember that mgmtPortConfig is used during the gateway configuration/initialization. Need to verify if switching interface wont break anything there.

Reply:
Can we close this discussion? The goal is not to add support for dynamic change of the interface at this point. The goal is to get rid of the requirement to manually set the vf rep.

Reply:
Yes let's close this discussion. We will have a bug here with device-plugin if the pool >= 2 devices and the DPU Host's ovnkube-node is restarted. But this case shouldn't matter for DPU yet.

Reply:
@bn222 Although we agreed to close the discussion, We will need to track this issue.

Reply:
Created https://issues.redhat.com/browse/NHE-563 to track this